### DISCUSSION TITLE
How to set the attack parameters of CW2? No matter how I change the parameters, I can't make CW2 successfully attack my model.

### DISCUSSION THREAD

**Discussion by leo-bleuming** [Trusted-AI/adversarial-robustness-toolbox#2542] [Q&A]
How to set the attack parameters of CW2? No matter how I change the parameters, I can't make CW2 successfully attack my model. 
System information:
OS win11
Python version 3.8.20
ART version or commit number 1.18.2
TensorFlow-gpu = 2.6.0 / Keras = 2.6.0

the cnn model I built:
[cnn_cifar10.txt](https://github.com/user-attachments/files/18144276/cnn_cifar10.txt)

**Comment by beat-buesser** [comment]
Hi @leo-bleuming Does you model return logits or probabilities?

**Reply by leo-bleuming** [reply]
I have provided detailed code and logs in issue #2541. Could you please help with the solution?
https://github.com/Trusted-AI/adversarial-robustness-toolbox/discussions/2541
